<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_fastopen">
       <property name="text">
        <string>Enable TCP fast open</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_localAddr">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Local Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_localPort">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_timeout">
       <property name="text">
        <string>Timeout</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_timeout">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_fastopen">
       <property name="text">
        <string>Fast Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Config Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_serverAddr">
         <property name="placeholderText">
          <string>Remote Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_serverPort">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_method">
       <property name="text">
        <string>Encryption</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_localAddr">
       <property name="text">
        <string>Local Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="comboBox_method">
         <item>
          <property name="text">
           <string notr="true">chacha20-ietf-poly1305</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-256-gcm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-192-gcm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-128-gcm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-256-cfb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-192-cfb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-128-cfb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-256-ctr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-192-ctr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">aes-128-ctr</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_serverAddr">
       <property name="text">
        <string>Server Address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_mode">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_mode">
       <item>
        <property name="text">
         <string notr="true">tcp_only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">tcp_and_udp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">udp_only</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
